### PR TITLE
[DCSFUS-40] QUERYING THE TRANSPORTATION DATABASE

### PR DESCRIPTION
**Pull request by bayooba**
This is my submission for the final task under the Databases module.
Attached is a screenshot of part of the SQL query statements. Thank you.

query_database

### CODE REVIEW

File: submissions/adebayoObafemi/database/query.sql [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+where accidents.survived = 1 and trips.point_of_embarkation = 'S';
+
+
+/* Question 5: Get the id, name and the total number of passengers
+ who paid a fare greater than $100 and above 
+ the age of 35 had siblings or spouses on board? */
+select id, full_name, count(full_name) from passengers
+left join trips
+on passengers.id = trips.passenger_id
+where passengers.age > 35 and trips.trip_fare > 100 and trips.no_of_siblings_or_spouses_aboard > 0;
```
Comment (Mayowa2020) [inline submissions/adebayoObafemi/database/query.sql:36]: Get the list of id, names and the total number of passengers  who paid a fare greater than $100 and above 
 the age of 35 had siblings or spouses on board as instructed  above,

**Review by Mayowa2020 — CHANGES_REQUESTED** [review]
Well done @bayooba, kindly make the requested changes.

File: submissions/adebayoObafemi/database/query.sql (hunk 1 above)
Comment (Mayowa2020) [reply submissions/adebayoObafemi/database/query.sql:41]: Hello @bayooba I would like to see your solutions to this last question. The total number and the list of passengers w

Comment (bayooba) [reply submissions/adebayoObafemi/database/query.sql:41]: Alright. I will do that and revert.

Comment (bayooba) [reply submissions/adebayoObafemi/database/query.sql:42]: I have included the result for the count in the SQL file. Attached is the screenshot of the solution to the first part for the question.
Screenshot (5)

Comment (bayooba) [reply submissions/adebayoObafemi/database/query.sql:42]: The updates have been included in the code already. I have attached another screenshot that captures the full list of passengers.

Screenshot (7)

**Review by Mayowa2020 — APPROVED** [review]
Well done @bayooba! **Approved.**

**Pull request merged**